src/cascadia/QueryExtension/AzureLLMProvider.cpp

```diff
@@ -79,6 +79,14 @@ namespace winrt::Microsoft::Terminal::Query::Extension::implementation
 
     winrt::Windows::Foundation::IAsyncOperation<Extension::IResponse> AzureLLMProvider::GetResponseAsync(const winrt::hstring& userPrompt)
     {
+        auto cancelation_token{ co_await winrt::get_cancellation_token() };
```

Comment:
this is very different from the camel case naming convention we use _everywhere else_...